fontforge/python.c

```diff
@@ -2625,14 +2589,172 @@ static PySequenceMethods PyFFContour_Sequence = {
     PyFFContour_Concat,		/* concat */
     NULL,			/* repeat */
     PyFFContour_Index,		/* subscript */
-    PyFFContour_Slice,		/* slice */
+    NULL,			/* slice */
     PyFFContour_IndexAssign,	/* subscript assign */
-    PyFFContour_SliceAssign,	/* slice assign */
+    NULL,			/* slice assign */
     PyFFContour_Contains,	/* contains */
     PyFFContour_InPlaceConcat,	/* inplace_concat */
     NULL			/* inplace repeat */
 };
 
+static PyObject *PyFFContour_Sub( PyObject *self, PyObject *key ) {
+    PyFF_Contour *cont = (PyFF_Contour *) self;
+    PyFF_Contour *ret;
+    Py_ssize_t start, end, step, len;
+    int i;
+
+    if ( PyInt_Check(key)) {
+        return PyFFContour_Index(self, PyNumber_AsSsize_t(key, PyExc_IndexError));
+    }
+    if ( !PySlice_Check(key) ) { 
+	PyErr_Format(PyExc_IndexError, "Contour indexed by integer only");
+	return( NULL );
+    }
+#if PY_VERSION_HEX > 0x03060100
```

Comment:
The docs explain the chain of support. As it turns out any recent version of python 2 or 3 will do pretty much the same thing underneath these functions because the new implementations were back-ported.